xAPI.md

```diff
@@ -3225,8 +3225,14 @@ Statements are filtered.
 * Activity Objects contain Language Map Objects for name, description and interaction components. 
 The LRS MUST return only one language in each of these maps. 
 
-* Verb Objects contain Language Map Objects for Display. 
-The LRS SHOULD* return only one language in this map. 
+* The LRS MAY maintain canonical versions of language maps for any language map used in an object
```

Comment:
I'd prefer not to introduce the concept of "object which has an id", but I'm not totally sure where to go with that.